spec/unit/outputs/elasticsearch_spec.rb

```diff
@@ -870,7 +890,7 @@
   end if LOGSTASH_VERSION > '6.0'
 
   context 'handling elasticsearch document-level status meant for the DLQ' do
-    let(:options) { { "manage_template" => false } }
```

Comment:
Maybe is best to use the `false` instead of `'false'` to keep the same pattern used in `manage_template`

Reply:
Same as https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1111#discussion_r1086724547

Reply:
oks